metabrainz/templates/index/about.html

```diff
@@ -3,42 +3,44 @@
 {% block title %}{{ _('About') }} - MetaBrainz Foundation{% endblock %}
 
 {% block content %}
-  <h1 class="page-title">{{ _('About the MetaBrainz Foundation') }}</h1>
+  <h1 class="page-title"><img src="{{ url_for('static', filename='img/logos/metabrainz.svg') }}" class="hidden-xs"></h1>
+
+     <h2 class="page-title">{{ _('About the MetaBrainz Foundation') }}</h2>
 
   <div class="row">
     <div class="col-sm-9">
       <p>
-        {{ _('The MetaBrainz Foundation is a 501(c)(3) tax-exempt non-profit founded in
-        December 2004 and based in San Luis Obispo, California, that believes in
-        free, open access to data.') }}
+        {{ _('The MetaBrainz Foundation is a global community creating and maintaining 
+        an open encyclopedia of music and arts metadata. Beginning as a resource for music 
+        collectors we currently establish collaborations between volunteers, artists, and 
+        industry leaders. Since it’s founding it has grown into a global community of 
+        <a href="%(editors_url)s">hundreds of thousands of editors</a> 
```

Comment:
All the placeholders like `%(editors_url)s` need to have a value defined to be filled in when the template is rendered. This can be done by passing them as an argument to the `_` (aka `gettext`) function that wraps the whole string.

```python
{{ _('A string of text with some placeholders inside. Like %(this_example)s...',
     this_example="this string") }}
```